sentry-core/src/main/java/io/sentry/core/Scope.java

```diff
@@ -188,15 +199,26 @@ public Scope clone() throws CloneNotSupportedException {
 
     final Map<String, Object> extraRef = extra;
 
-    Map<String, Object> extraClone = new HashMap<>();
+    Map<String, Object> extraClone = new ConcurrentHashMap<>();
 
     for (Map.Entry<String, Object> item : extraRef.entrySet()) {
       if (item != null) {
         extraClone.put(item.getKey(), item.getValue());
       }
     }
 
-    clone.extra = extraClone;
```

Comment:
Good catch. It's a a mistake. I didn't review anything, just pushed not to forget to fix that later, builds also broken etc

Reply:
ideally, I'd like to use `org.apache.commons.lang3.SerializationUtils` for a deep copy, but dependencies are not allowed.

manual deep copying is very error-prone, but well :(

I even did my own deep copy of `enums`, they are immutable by default, but if we extend and add mutable fields to them, well, they will become shallow copy and I don't see how to protect that behavior with unit test, maybe with reflection to all the current fields, but it's too much work for something that might never change.

/rant :D

CI is broken because of linting - ModifiedButNotUsed

all of this for @bruno-garcia revert this line

Reply:
Will do